packages/shared/src/variants/quantum.ts

```diff
@@ -310,4 +310,7 @@ export const quantumVariant: Variant<NewBadukConfig> = {
       komi: 7.5,
     };
   },
+  // quantum uses its own board, so we shouldn't use
+  // Baduk.uiTransform
+  uiTransform: undefined,
```

Comment:
Maybe we can use the default board as child component of the quantum go board. But that's for the future anyway.

Reply:
Oh yeah! Tbh I didn't even look at the impl of QuantumBoard when doing this, but migrating to DefaultBoard sounds like the right path!